updateDates.py

```diff
@@ -9,22 +9,34 @@
 
 
 filePath = "sessionize/app/src/main/assets/schedule.json"
+outputPathAndroid = "sessionize/app/src/main/assets/schedule.json"
+outputPathiOS = "iosApp/iosApp/schedule.json"
```

Comment:
Should this be customizable like outputPathAndroid is?

Reply:
For simplicity I made it so that if you specify a custom path, it will only write that one file. Figured adding multiple paths might be too much